src/js/page_sections/Menu/Menu.js

```diff
@@ -192,4 +191,10 @@ const MobileFooter = injectIntl(({intl}) => (
   </div>
 ))
 
-export default withSiteData(injectIntl(Menu));
+Menu.childContextTypes = {
```

Comment:
Did you mean PropTypes instead of childContextType? If not, can you explain how're you're using this?

Reply:
Also I like your destructuring approach at the top of the render function so it's clear what props are being used. With propTypes, it's probably redundant, but I like extra obvious things. could we do that in Menu.js as well. So
```
const { menuIsOpen, toggleMenu, navigation } = this.props;
```